### PR TITLE
Run ii 94 x v1

### PR DESCRIPTION
**Pull request by garbersc**
added reprocessed RunE xml 

and all use of genparticles is in try-catch structures that supress error due to missing genparticle information in files used by BaconJets
keep it or set it back?

### CODE REVIEW

**Comment by karavdin** [comment]
Might be useful for the future too. Thanks!

**Comment by raggleton** [comment]
What was the issue with the genInfo stuff? Does it just randomly fail?

**Comment by garbersc** [comment]
no, we had a ntuple processing run of the MC were the gen information was not saved due to a misset flag

File: core/src/EventHelper.cxx [hunk 1]
```diff
@@ -37,6 +37,8 @@ EventHelper::EventHelper(uhh2::Context & ctx_): ctx(ctx_), event(0), pvs(false),
 #define IMPL_SETUP(name, type) void EventHelper::setup_##name (const std::string & bname) {\
     if(bname.empty()) return; \
     name = true; \
+    std::cout<<"going to declare "<<bname<<std::endl; \
```
Comment (raggleton) [inline core/src/EventHelper.cxx]: Please remove

**Comment by karavdin** [comment]
Hi @garbersc,
Something is broken in this PR. After the switch to it I can't run MC samples any more. However it was running fine on the same sample before the switch.
The error:
Event: error for member of type 'GenInfo' at index 18446744073709551615: get: index out of bounds, i.e. the handle used is invalid
Appears  for following line:
double pt_hat = ev.genInfo->binningValues()[0]

Did you test it on MC?

File: core/src/EventHelper.cxx [hunk 2]
```diff
@@ -37,6 +37,8 @@ EventHelper::EventHelper(uhh2::Context & ctx_): ctx(ctx_), event(0), pvs(false),
...
+    if(bname == "genInfo") return; \
```
Comment (raggleton) [inline core/src/EventHelper.cxx]: @karavdin I'd guess this is why it stopped working on MC for you?

**Pull request merged**